elk/train.py

```diff
@@ -30,14 +31,16 @@ def train(args):
         mode=args.mode,
         num_data=args.num_data,
     )
+    permutation = get_permutation(hidden_states)
+    normalized_hidden_states = normalize(hidden_states, permutation)
```

Comment:
I guess you are right; it is cleaner when the normalization happens outside of the get_hidden_states function. Thanks